### ISSUE TITLE
Did you accidentally change back the colours of the tab bar?

### ISSUE THREAD

**Issue by leo** [fv0/native-ui#41]
Please correct me, if I'm wrong. But didn't I fix the colours for the inactive tab bar in #37? Because now, they look wrong again. Here's a comparison to the native tabs:

**Your theme:**

<img width="891" alt="screen shot 2016-01-24 at 12 10 22" src="https://cloud.githubusercontent.com/assets/6170607/12535908/b8c349b6-c293-11e5-9c7f-5c71bf90f08b.png">

**Native tabs:**

<img width="1035" alt="screen shot 2016-01-24 at 12 12 35" src="https://cloud.githubusercontent.com/assets/6170607/12535911/c6f83e6a-c293-11e5-9d40-419fafd196c6.png">

**Comment by fv0** [comment]
Yes, that's true. The `.is-blurred` selector needed to be applied to a whole lot of other elements too and for ease of maintenance I moved the selector to the global workspace selector that gives everything a reduced opacity. I also turned out that the active and inactive background colors for the tabs didn't contrast enough for my taste. The theme will never appear and behave 100% accurate. OSX displays some things in a way that just isn't good design so I made some tweaks here and there to make sure _working_ with this theme feels great.

**Comment by leo** [comment]
WOW. Saying that OS X has parts that aren't perfectly designed, is a really big thing to say. :smile_cat: I mean, those guys are the prime of what's currently possible, if you ask me.

However it's your theme, so we'll have to live with your changes... :see_no_evil:

**Comment by leo** [comment]
Okay. I probably won't be able to convince you, but nevertheless, I'd like to know the reason why you've completely removed the blurred window state. :blush:

I'm just really interested in the answer behind that idea. Because in my mind, it simply makes much more sense to treat the work of making it look like the native UI with a higher priority than making it look "better". What do you think about this? Why are you making it look different than the rest of the OS?

I mean, I understand that you don't like certain things within the UI of El Capitan. But isn't the purpose of this theme to look exactly like the **native(-)ui**? :see_no_evil:

**Comment by fv0** [comment]
I totally see your point. It's not that I don't like it, it just wasn't implemented too well yet. That's why I kicked it out for now. Your solution worked for the tabs but in other parts of the interface I had to manually adjust border colors and this just became a bit too "hacky". I've started working on a better approach to it, because I agree with your thoughts about the native behaviour and they make sense.
